src/app/page.js

```diff
@@ -1,3 +1,9 @@
+import DemoHero from "@/app/_components/DemoHero/DemoHero";
```

Comment:
You can create index.js file in DemoHero folder to reexport to avoid /ComponentName/ComponentName, e.g. `export { default } from './DemoHero'`